components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -65,87 +75,93 @@ const ContributorProfilePage = ({
   });
 
   const { chart } = useTopicContributorCommits(githubName, "*", repositories);
+  const prsMergedPercentage = getPercent(prTotal, prMerged || 0);
+  const isLoaded = !loading && !error;
 
   return (
     <div className=" w-full">
-      <ContributorProfileHeader avatarUrl={githubAvatar} />
-      <div className="pt-24 px-4 md:px-10 lg:px-16 flex flex-col lg:flex-row lg:gap-40 w-full overflow-hidden justify-between">
-        <div className="flex flex-col min-w-[270px] gap-4 ">
-          <div className="pb-6 border-b">
-            <Title className="!text-2xl !text-light-slate-12" level={3}>
-              {githubName}
-            </Title>
-            <span className="text-light-slate-11 text-sm">{`@${githubName}`}</span>
-          </div>
-          <div>
-            <p className="mb-4">Languages</p>
-            <CardHorizontalBarChart languageList={languageList} />
-          </div>
-        </div>
-        <div className="flex-1">
-          <div>
-            <Title className="!text-light-slate-12 !text-xl" level={4}>
-              Contribution Insights
-            </Title>
-          </div>
-          <div className="bg-white mt-4 rounded-2xl border p-4 md:p-6">
-            <div className=" flex flex-col lg:flex-row gap-2 md:gap-12 lg:gap-16 justify-between">
-              <div>
-                <span className="text-xs text-light-slate-11">PRs opened</span>
-                {openPrs ? (
-                  <div className="flex lg:justify-center md:pr-8 mt-1">
-                    <Text className="!text-lg md:!text-xl lg:!text-2xl !text-black !leading-none">{openPrs} PRs</Text>
-                  </div>
-                ) : (
-                  <div className="flex justify-center items-end mt-1"> - </div>
-                )}
+      {loading && <>Loading...</>}
+      {error && <>An error has occured...</>}
+      {isLoaded && (
+        <>
+          <ContributorProfileHeader isOpenSaucedUser={!!user} githubName={githubName} avatarUrl={githubAvatar} />
```

Comment:
this should be `isOpenSaucedUser={!!user.is_open_sauced_member}`?

Reply:
The check for `user.is_open_sauced_member` is meant for member's of the org... Per our discussions in the engineering sync yerterday,  once  a user is returned from the api call, they are opensauced users, so we don't have to do additional checks for `is_open_sauced_member` 🍕

Reply:
That is not correct, we have users which have `is_open_sauced_member` false. We will always have users like that because we cannot control who owns a repo.

Reply:
We should change this prop to `isVerified={!!user}` or `isConnected={!!user}` so its not so specific. Verified/connected just means they've connected their GitHub account